dbt/adapters/sqlserver/connections.py

```diff
@@ -68,7 +68,7 @@ def _connection_keys(self):
         
     
         return 'server', 'database', 'schema', 'port', 'UID', \
-                 'authentication', 'encrypt'
+               'client_id','authentication', 'encrypt'
```

Comment:
ClientId was not displayed in `dbt debug` window for `Auth=ServicePrincipal`
![image](https://user-images.githubusercontent.com/67547199/94122412-2348fe80-fe70-11ea-89ff-e6782cbc19c3.png)